package.json

```diff
@@ -84,6 +84,7 @@
     "require-ancestors": "^1.0.0",
     "require-in-the-middle": "^3.1.0",
     "semver": "^5.5.0",
+    "set-cookie-serde": "github:qard/set-cookie-serde#implementation",
```

Comment:
This PR is marked as "In Review", but I guess it should still be "In Progress" until this dependency have been published right?